packages/list/ListEditing.js

```diff
@@ -41,28 +38,13 @@ ListEditing.Prototype = function() {
     // current position of the current node
     if (offset === 0) {
       if (text.length === 0) {
-        // if we hit return on an already empty list item, it should transform into
+				// if we hit return on an already empty list item, it should transform into
         // a paragraph
-        var index = container.getChildIndex(parentList); // index of current list
-        // create a paragraph node
-        var defaultType = tx.getSchema().getDefaultTextType();
-        id = uuid(defaultType);
-        newNode = tx.create({
-          id: id,
-          type: defaultType,
-          content: ""
-        });
-        // show the paragraph node
-        container.show(id, index+1);
-        // remove the current empty list item from the list
-        tx.update([node.parent, 'items'], {delete: {offset: parentList.items.indexOf(node.id)}});
-        newPath = [id, 'content'];
-        // trandfer the cursor to the paragraph
-        selection = tx.createSelection({
-          type: 'property',
-          path: newPath,
-          startOffset: 0
-        });
+				// set the needed arguments and call mergeListItems behavior
+				args.direction = 'left';
+				args.path = path;
+				args.node = parentList;
+				args = this.mergeListItems(tx, args);
```

Comment:
Yes. Should we put it in the same file or something like ListUtils.js? Both editingBehavior and ListCommand will use these functions.